src/languageHandler.js

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
```

Comment:
Fixed. @MTRNord , should we share (c) for the languageHandler? Or do do you prefer to have it only in your name? It's up to you! You will see that I've put "NordGedanken and Cooperative EITA", but it can be whatever you want, since you really made it!